app/Http/Controllers/Admin/InquiriesController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Inquiry;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class InquiriesController extends Controller
+{
+    private $inquiry;
+
+    public function __construct(Inquiry $inquiry, User $user)
+    {
+        $this->inquiry = $inquiry;
+        $this->user = $user;
```

Comment:
Please declare a private $user variable

Reply:
Thank you so much!
I added private $user variable, and push it.